code/PostProcessing/ComputeUVMappingProcess.cpp

```diff
@@ -51,10 +51,10 @@ using namespace Assimp;
 
 namespace {
 
-    const static aiVector3D base_axis_y(0.0,1.0,0.0);
```

Comment:
What is the advantage?

Reply:
static in an anonymus namespace is redundant